src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a .css file, not a .scss file. The browser does not understand SCSS, it only understands CSS. The SCSS files are compiled into a CSS file which is then linked in the HTML. So, you need to link the compiled CSS file here.